csunplugged/tests/topics/loaders/test_curriculum_areas_loader.py

```diff
@@ -64,6 +61,14 @@ def test_curriculum_areas_loader_correct_name_value(self):
             "Maths",
         )
 
+    def test_curriculum_areas_loader_missing_data(self):
+        config_file = "missing-data.yaml"
+        area_loader = CurriculumAreasLoader(structure_filename=config_file, base_path=self.base_path)
+        self.assertRaises(
```

Comment:
We've got a mix of this style and the context manager (```with self.assertRaises()```) style. Not a major issue, but should we decide on a standard moving forward?

Reply:
Could do, but not really a major issue (I haven't seen the with style yet in our repo, but I presume it's there somewhere). Some thing could be said about Mock decorators vs `with` vs function calls. All are standard uses, I'm fine with this. Thoughts?